doc/source/train/api/api.rst

```diff
@@ -35,13 +35,13 @@ PyTorch
 .. autosummary::
     :toctree: doc/
 
+    ~train.torch.get_device
```

Comment:
I feel there are too many apis here, can we separate them into basic(`get_device`, `prepare_model`, `prepare_data_loader`) and advanced(others)?